src/routes/callback/+server.ts

```
@@ -0,0 +1,37 @@
+  const code = url.searchParams.get('code');
+  const next = url.searchParams.get('next') || '/reports';
+
+  if (code) {
+    const { data, error } = await supabase.auth.exchangeCodeForSession(code);
+
+    if (error) {
+      console.error('Auth error:', error);
+      throw redirect(303, '/signin?error=auth');
+    }
+
+    if (data.user && data.user.email) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)